CONTRIBUTING.md

```diff
@@ -27,3 +27,29 @@ for details, such as:
 * Development setup and testing your changes
 * Submitting a pull request and finding a reviewer
 
+To build and install the whole project from the source distribution,
+you need additional tools installed in your system including:
+
+* JDK 8 or later
+* Python 3
```

Comment:
Would you clarify which Python 3? (3.4, 3.7, etc.?)

Reply:
Any idea which versions are supported/required?

Reply:
done